src/components/Map/PlantingSiteMap.tsx

```diff
@@ -13,10 +13,15 @@ export type PlantingSiteMapProps = {
 
 export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Element {
   const { siteId } = props;
+  const theme = useTheme();
   const [snackbar] = useState(useSnackbar());
   const [token, setToken] = useState<string>();
   const [mapOptions, setMapOptions] = useState<MapOptions>();
 
+  const getFillColor = useCallback(() => {
+    return theme.palette.TwClrBaseWhite || 'white';
```

Comment:
this is still temporary but looks closer to mocks, previous blue/green/yellow was very wrong. the '||' white is because fillColor for map needs to be string, palette color is string | undefined.